src/Microsoft.AspNetCore.Cors/Infrastructure/LoggerExtensions.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Cors.Infrastructure
+{
+    internal static class LoggerExtensions
+    {
+        private static readonly Action<ILogger, Exception> _failedToSetCorsHeaders;
+
+        static LoggerExtensions()
+        {
+            _failedToSetCorsHeaders = LoggerMessage.Define(
+                LogLevel.Warning,
+                1,
```

Comment:
There should be other logging events from Cors, this new one should be after them

Reply:
Ahh, didn't notice it was sitting under the Internal ns.